### PR TITLE
Blog: Intro FlowFuse Dedicated

### PR DESCRIPTION
**Pull request by MarianRaphael**
## Description

Blog article about FlowFuse Dedicated

## Related Issue(s)

closes https://github.com/FlowFuse/website/issues/1891 https://github.com/FlowFuse/website/issues/1815

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Comment by Hasmin-AC** [comment]
Does it make sense to have a separate form to inquire/get started with Dedicated cloud? Maybe include some specific set up questions? @ZJvandeWeg

**Comment by ZJvandeWeg** [comment]
A separate form is definitely needed and a traceable way of capturing prospects.

The forms contents should include all business details, and some indication it's not cheap.

**Comment by Hasmin-AC** [comment]
pricing details etc can go in the landing page along with the form. This blog can link to that page for more details instead

File: src/blog/2024/04/flowfuse-dedicated.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Introducing FlowFuse Dedicated
```
Comment (Hasmin-AC) [inline src/blog/2024/04/flowfuse-dedicated.md:2]:
```suggestion
title: Introducing FlowFuse Dedicated Cloud
```

File: src/blog/2024/04/flowfuse-dedicated.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- **Regional Deployment Flexibility:** FlowFuse Dedicated can be deployed in the region of your choice, ensuring data residency requirements are met while optimizing performance by reducing latency. FlowFuse Dedicated can be deployed in 30+ regions.
+- **Enterprise-Grade Features:** Enjoy access to all enterprise features and configurations as your own FlowFuse Administrator, enabling you to manage multiple teams effectively.
+- **Comprehensive Compliance:** Meet complex compliance standards with ease, thanks to full data and source code IP isolation.
+
+FlowFuse Dedicated represents a paradigm shift in Node-RED offerings, a solution that not only meets the demanding requirements of today's enterprises in terms of speed, efficiency, and productivity but also navigates the complex terrain of compliance and data security with unparalleled finesse. If you are intrested in FlowFuse Dedicated, [contact us](https://flowfuse.com/contact-us/).
```
Comment (Hasmin-AC) [inline src/blog/2024/04/flowfuse-dedicated.md:28]: link to new landing page as the CTA to contact

Comment (Yndira-E) [reply src/blog/2024/04/flowfuse-dedicated.md:29]: Here's the PR for that page:https://github.com/FlowFuse/website/pull/1954

File: src/blog/2024/04/flowfuse-dedicated.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
```
Comment (Hasmin-AC) [inline src/blog/2024/04/flowfuse-dedicated.md:29]:
```suggestion
FlowFuse Dedicated represents a paradigm shift in Node-RED offerings, a solution that not only meets the demanding requirements of today's enterprises in terms of speed, efficiency, and productivity but also navigates the complex terrain of compliance and data security with unparalleled finesse. If you are interested in FlowFuse Dedicated, [contact us](https://flowfuse.com/dedicated).
```

File: src/blog/2024/04/flowfuse-dedicated.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  - node-red
+  - flowfuse
+  - dedicated
+---
+
+Today, we are excited to officially announce FlowFuse Dedicated, a new way to use our enterprise platform as a single-tenant SaaS offering. This new offering provides all of the benefits of FlowFuse enterprise, with an added focus on data residency, isolation, and private networking to meet compliance needs.
+
+<!--more-->
+
+At FlowFuse, we cater to a diverse spectrum of clients, ranging from burgeoning startups to formidable global enterprises. It's clear that a one-size-fits-all approach to deployment doesn't align with the varied needs of our customer base. The stakes for compliance and security have never been higher. Even sectors not traditionally bound by stringent regulatory frameworks are finding compliance to be a critical factor impacting productivity and profitability.
```
Comment (ZJvandeWeg) [inline src/blog/2024/04/flowfuse-dedicated.md:19]: Maybe mention:

> FlowFuse's product already has flexible deployment options, ranging from FlowFuse Cloud, to a cloud of your chosing, to yours own hardware. Allowing our customers to achieve complience even in the stringent regulatory frameworks.

And remove the 

> The stakes for compliance and security have never been higher.

It's a bit much IMO

File: src/blog/2024/04/flowfuse-dedicated.md (hunk 2 above)
Comment (ZJvandeWeg) [reply src/blog/2024/04/flowfuse-dedicated.md:29]:
```suggestion
FlowFuse Dedicated represents a paradigm shift in Node-RED offerings, a solution that not only meets the demanding requirements of today's enterprises in terms of speed, efficiency, and productivity but also navigates the complex terrain of compliance and data security with unparalleled finesse. [Learn more about FlowFuse Dedicated](https://flowfuse.com/dedicated/).
```

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: src/blog/2024/04/flowfuse-dedicated.md (hunk 4 above)
Comment (MarianRaphael) [reply src/blog/2024/04/flowfuse-dedicated.md:19]:
```suggestion
At FlowFuse, we cater to a diverse spectrum of clients, ranging from burgeoning startups to formidable global enterprises. It's clear that a one-size-fits-all approach to deployment doesn't align with the varied needs of our customer base. FlowFuse's product already has flexible deployment options, ranging from FlowFuse Cloud, to a cloud of your chosing, to yours own hardware. Allowing our customers to achieve complience even in the stringent regulatory frameworks. Even sectors not traditionally bound by stringent regulatory frameworks are finding compliance to be a critical factor impacting productivity and profitability.
```

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**